.github/containers/nox-cross-arch/arm64-ubuntu-20.04-python-3.12.Dockerfile

```diff
@@ -0,0 +1,33 @@
+# License: MIT
```

Comment:
Can't we parameterize this file instead using [docker build variables](https://docs.docker.com/build/building/variables/)?

Reply:
I think it should be possible. I'll give it a try

Reply:
It works fine, suggestion addressed